docs/content/developers/writing-style-guide.md

```diff
@@ -0,0 +1,222 @@
+# Writing Style Guide
+
+This page formalizes the writing conventions we aspire to use in the documentation.  
+It’s a living statement of intent and a reference for all contributors.
+
+## Voice and Tone
+
+In the spirit of the [Code of Conduct](../users/code-of-conduct.md), we want to be clear and encouraging for everyone that bothers to read DDEV’s documentation, rewarding the time and attention they choose to give to it.
+
+### Beginner-Friendly, Expert-Compatible
+
+Write so a DDEV beginner can follow your guidance and a DDEV veteran could use the same content as a reference.
+
+| Write This 👍 | Not This ❌
+| -- | --
+| You can run `ddev restart` to apply changes you’ve made to your global `~/.ddev/global_config.yaml` or per-project `.ddev/config.yaml`. | Just `ddev restart` to apply YAML config changes.
+
+### Be Efficient and Direct
+
+Brevity makes for good documentation!
+
+Few read the docs from start to finish like a book, and explanation can be a barrier to learning. Get to the point, avoiding parentheticals and mid-sentence notes that interrupt the main flow.
+
+Omit extraneous explanation or decorative language that doesn’t help the reader. Instructions don’t need to include “please”. Provide some context for anchor links without overloading them to hinder flow.
+
+| Write This 👍 | Not This ❌
+| -- | --
+| Run `ddev start` and launch the site in a browser. | Please run `ddev start`, then launch the site in a browser.
```

Comment:
```suggestion
| Run `ddev start` and launch the site in a browser. | Please run `ddev start`, then use `ddev launch` to view the site in a browser.
```

Reply:
This may be an awful example to start with—added `ddev launch` to the _ideal_ example as well, since your suggested improvement only made the “Not This” example more useful.

```suggestion
| Run `ddev start`, then run `ddev launch` to open the site in a browser. | Please run `ddev start`, then use `ddev launch` to view the site in a browser.
```

Reply:
This example is really just illustrating how not to start a sentence with “Please”, which is probably insulting to anyone with a pulse. Might be better to get rid of it or come up with another one.

Reply:
Your choice here. It's hard to review these tables in a PR, so gets confusing.

Reply:
I’ll come back to this and try for a new example. In the meantime, I added a screenshot to the PR description since the illustrations (via table and `!!!tip` examples) are a big part of it.

Reply:
@rfay 
![image](https://user-images.githubusercontent.com/8931305/194672710-7b34524d-2b13-49d7-937e-31872087dbf7.png)

Clicking on this button will give you a more Visual view of the PR.